broker.go

```diff
@@ -1088,11 +1058,12 @@ func (c *offsetCoordinator) commit(
 
 		// get a copy of our connection with the lock, this might establish a new
 		// connection so can take a bit
-		conn, err := c.getConnection()
+		conn, err := c.broker.coordinatorConnection(c.conf.ConsumerGroup)
```

Comment:
I suspect this situation is strictly better. We're not using any more connections than we were -- previously it was still subject to the connection limit, it would just "pin" that connection forever to the coordinator even if it was totally idle. I believe the old state would, in general, use more connections because we're guaranteed to have at least one permanently idle one.